fe/fe-core/src/main/java/org/apache/doris/common/proc/BackendsProcDir.java

```diff
@@ -51,7 +51,7 @@ public class BackendsProcDir implements ProcDirInterface {
             .add("BePort").add("HttpPort").add("BrpcPort").add("LastStartTime").add("LastHeartbeat").add("Alive")
             .add("SystemDecommissioned").add("ClusterDecommissioned").add("TabletNum")
             .add("DataUsedCapacity").add("AvailCapacity").add("TotalCapacity").add("UsedPct")
-            .add("MaxDiskUsedPct").add("Tag").add("ErrMsg").add("Version").add("Status")
+            .add("MaxDiskUsedPct").add("Tag").add("ErrMsg").add("Version").add("Status").add("RemoteUsedCapacity")
```

Comment:
Better to add `RemoteUsedCapacity` right after `DataUsedCapacity`

Reply:
Some unit tests and regression tests rely on the subscript of these data. If `RemoteUsedCapacity` is inserted into the middle, the tests will fail.

Reply:
done